### PR TITLE
Paging/limit support in MRY

### PR DESCRIPTION
**Pull request by appaquet**

### CODE REVIEW

File: mry-core/src/main/scala/com/wajam/mry/execution/Value.scala [hunk 1]
```diff
@@ -11,18 +11,14 @@ trait Value extends Object with OperationSource {
 
   def equalsValue(that: Value): Boolean = this == that
 
-  def isNull = this.equalsValue(NullValue.NULL_VALUE)
+  def isNull = this.equalsValue(NullValue)
 }
 
-class NullValue() extends Value with Serializable {
+class NullValue extends Value with Serializable {
```
Comment (alexbergeron) [inline mry-core/src/main/scala/com/wajam/mry/execution/Value.scala:17]: Stupid question, but could you avoid implementing a class and a object with only a NullValue object that extends Value with Serializable

**Comment by alexbergeron** [comment]
+1

File: mry-core/src/main/scala/com/wajam/mry/execution/Value.scala (hunk 1 above)
Comment (appaquet) [inline mry-core/src/main/scala/com/wajam/mry/execution/Value.scala:17]: I could just implement an object instead... Didn't see that  ;)

Comment (appaquet) [inline mry-core/src/main/scala/com/wajam/mry/execution/Value.scala:17]: Finally, would be shotgun surgery and would change the PR all over. Good for a future PR

**Pull request merged**